File: telegram-bot-bumblebee-core/src/main/java/com/github/bumblebee/command/start/StartCommand.java
package com.github.bumblebee.command.start;

import org.apache.commons.io.IOUtils;
import telegram.api.BotApi;
import telegram.domain.Update;
import telegram.domain.request.ParseMode;
import telegram.polling.UpdateHandler;

import java.io.IOException;
import java.io.InputStream;
import java.io.StringWriter;

/**
 * Provide info about supported features
 */
public class StartCommand implements UpdateHandler {

    private final BotApi botApi;
    private final String helpText;

    public StartCommand(BotApi botApi) {
        this.botApi = botApi;

        try (InputStream is = StartCommand.class.getClassLoader().getResourceAsStream("start.md")) {
            StringWriter sw = new StringWriter();
            IOUtils.copy(is, sw);
            this.helpText = sw.toString();
        } catch (IOException e) {
            throw new RuntimeException(e);
        }
    }

    @Override
    public boolean onUpdate(Update update) {

        botApi.sendMessage(update.getMessage().getChat().getId(), helpText, ParseMode.MARKDOWN, null,
                update.getMessage().getMessageId(), null);

        return true;
    }
}

File: telegram-bot-bumblebee-core/src/main/kotlin/com/github/bumblebee/command/start/StartCommand.kt
package com.github.bumblebee.command.start

import com.github.bumblebee.bot.consumer.UpdateHandler
import com.github.telegram.api.BotApi
import com.github.telegram.domain.ParseMode
import com.github.telegram.domain.Update
import org.apache.commons.io.IOUtils
import org.springframework.core.io.ClassPathResource
import org.springframework.stereotype.Component
import java.nio.charset.StandardCharsets

@Component
class StartCommand(val botApi: BotApi) : UpdateHandler {
    private val helpText: String by lazy {
        ClassPathResource("start.md").inputStream.use { IOUtils.toString(it, StandardCharsets.UTF_8) }
    }

    override fun onUpdate(update: Update): Boolean {
        botApi.sendMessage(
            chatId = update.message!!.chat.id,
            text = helpText,
            parseMode = ParseMode.MARKDOWN
        )
        return true
    }
}